test/WebSites/BasicWebSite/wwwroot/javascript/jquery.validate.unobtrusive.3.2.6.js

```diff
@@ -0,0 +1,416 @@
+/*!
```

Comment:
@pranavkm did a thing not too long ago where we check in `_grunt.js` (note the `_`) so that it normally isn't run. But when we want to update things it's trivial: rename the file, update the version specs in it, run grunt, and rename the file back. We should do that here.